backend/pong/ws/tournament/async_db_serivice.py

```
@@ -122,23 +215,18 @@ def update_tournament_status(
-                "DoesNotExist": f"Tournament object does not exists. Details: {str(e)}."
-            }
+            handle_does_not_exist_error(
+                e, "Tournament object does not exists."
+            )
         )
 
     except DatabaseError as e:
-        logger.error(f"DatabaseError: {e}")
         return UpdateTournamentResult.error(
-            {"DatabaseError": f"Failed to create account. Details: {str(e)}."}
+            handle_database_error(e, "Failed to update tournament.")
```

Ruff (0.8.2)

Undefined name `handle_database_error`

(F821)